### PR TITLE
Jordan/action modal footer

### PR DESCRIPTION
**Pull request by jbibla**
**Description:**
- cleaned up a bunch of action modal styles 
- specifically focussed on mobile improvements - the buttons aren't fixed at the bottom on phones anymore and should result in a less glitchy action modal experience on iOS and Android

Screen Shot 2020-05-02 at 4 36 41 PM
Screen Shot 2020-05-02 at 4 35 32 PM
Screen Shot 2020-05-02 at 4 30 19 PM

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -10,17 +10,6 @@
   >
     <template v-if="validator.operatorAddress" slot="managed-body">
       <div class="button-container">
-        <button
```
Comment (faboweb) [inline src/components/staking/PageValidator.vue:13]: why did you remove this?

Comment (jbibla) [reply src/components/staking/PageValidator.vue:13]: this button doesn't really make sense. the validator profile is not _always_ visited after the validator list. why should the button say "back to validators" if you visit a validator profile from your portfolio? i don't like this kind of exception — we don't do this for any other routes. 

we could add a proper back button (`-1`) — but i don't think `Pages` actually need a back button.

Comment (jbibla) [reply src/components/staking/PageValidator.vue:13]: it is unrelated to this PR. sorry about that.

Comment (faboweb) [reply src/components/staking/PageValidator.vue:13]: We decided on having this in the past. Maybe change the wording? This is app focused design where you can't go back to the validators in one click.

Comment (Bitcoinera) [reply src/components/staking/PageValidator.vue:13]: Actually I had on my TODO list transforming this button functionality so it could be `Back to Validators/ Back to Portfolio` depending where you were visiting from.

But yes, I don't think deleting it altogether is the way to go either

Comment (jbibla) [reply src/components/staking/PageValidator.vue:13]: > This is app focused design where you can't go back to the validators in one click.

but i think you can do this now with our mobile menu

**Comment by Bitcoinera** [comment]
I am not going to approve this PR until you bring back the `Back to Validators` button, sorry.

Also it is out of scope for this PR

File: src/components/staking/PageValidator.vue (hunk 1 above)
Comment (faboweb) [reply src/components/staking/PageValidator.vue:13]: true

**Comment by jbibla** [comment]
ok i added it back (despite how much i dislike this button)

**Comment by Bitcoinera** [comment]
> ok i added it back (despite how much i dislike this button)

oh no, sorry about that. Don't worry, I will fix the logic 😉

**Comment by faboweb** [comment]
@Bitcoinera we will probably remove the button. But we discussed that radical UI changes should go in separate PRs

**Comment by Bitcoinera** [comment]
> @Bitcoinera we will probably remove the button. But we discussed that radical UI changes should go in separate PRs

Nooo. It can be "smart" and just take you back to where you came from
